backend/src/main/Index.ts

```diff
@@ -0,0 +1,4 @@
+import { App } from './App';
+
+const app: App = new App();
+app.init().then();
```

Comment:
Ahh, curious -- is there a difference without the empty `.then` block?

Reply:
Oh should be no difference, I was getting like a warning that the promise returned is ignored so I just added it